test/features/help.js

```diff
@@ -0,0 +1,59 @@
+const sinon = require("sinon");
+const suppressLogs = require("mocha-suppress-logs");
+const expect = require("chai").expect;
+const rewire = require("rewire");
+
+const MockController = require("../mocks/controller");
+
+const helpFeature = rewire("../../features/help");
+
+describe("features / help", () => {
+  suppressLogs();
+  let controller;
+
+  describe("responds to events", () => {
+    beforeEach(async () => {
+      controller = new MockController({});
+      helpFeature.__set__(
+        "respondToHelp",
+        sinon.spy(helpFeature.__get__("respondToHelp"))
```

Comment:
Generally if a function is part of the system under test it shouldn't be mocked or spied on, including private functions that aren't exposed outside of a module. Instead, those should reachable somehow through the public interface. 

In this case, I'd remove the `rewire` setup and instead verify what the `respondToHelp` function does. Testing that a private method was invoked is more of an implementation detail -- this test would fail if the function was renamed or inlined into the `controller.hears` call.